src/main/java/edu/hawaii/its/api/util/JsonUtil.java

```diff
@@ -28,6 +28,7 @@ public static <T> T asObject(final String json, Class<T> type) {
         try {
             result = new ObjectMapper().readValue(json, type);
         } catch (Exception e) {
+            logger.error("Error: " + type);
```

Comment:
I like the addition of putting the type in the error. If you could, combine the two `logger.error` into one, thanks.